renderer/src/App.tsx

```diff
@@ -1,34 +1,37 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
 import './App.css'
-import {
-  Link, Route, Routes
-} from 'react-router-dom'
-import { ActivityLog } from './components/ActivityLog'
-import Saturn from './Saturn'
-import { TotalJobsCompleted } from './components/TotalJobsCompleted'
 import { useEffect } from 'react'
-import OnboardingPage from './pages/Onboarding'
+import Onboarding from './pages/Onboarding'
+import WalletConfig from './pages/WalletConfig'
+import Saturn from './Saturn'
+// import { ActivityLog } from './components/ActivityLog'
+// import { TotalJobsCompleted } from './components/TotalJobsCompleted'
 
 const App = ():JSX.Element => {
+  useEffect(() => { document.title = 'Filecoin Station' })
   return (
-    <Routes>
-      <Route path="/" element={<OnboardingPage />} />
-      <Route path="/dashboard" element={<Dashboard />} />
-      <Route path="/saturn" element={<Saturn />} />
-    </Routes>
+    <Router>
+      <Routes>
+        <Route path="/" element={<Onboarding />} />
+        <Route path="/wallet" element={<WalletConfig />} />
+        <Route path="/dashboard" element={<Saturn />} />
+
+      </Routes>
+    </Router>
   )
 }
 
 export default App
 
-function Dashboard (): JSX.Element {
-  useEffect(() => { document.title = 'Filecoin Station' })
+// function Home (): JSX.Element {
+//   useEffect(() => { document.title = 'Filecoin Station' })
```

Comment:
is this ready for review?

Reply:
I think we can start reviewing it.
There are three things missing, two of them can be another PR
- e2e test failing
- activity log keyboard scroll needs to be reviewed in terms of behaviour
- skeleton loading missing

Reply:
Yes it is

Reply:
ah, I thought it wasn't ready because it contained commented out code. Do you think we could keep the demo activity log component, so that the app stays demoable after merging this PR?

Reply:
We can keep the component, but maybe moving it to `Station.tsx`?

Reply:
sure!

Reply:
1d0350e